docs/ChangeLog.md

```diff
@@ -1,3 +1,68 @@
+# 0.42.0
+Infrastructure:
+* Kotlin 1.9.0
+
+Deprecations:
+* deprecate: EXPOSED-84 Raise deprecation levels of API elements by @bog-walk in https://github.com/JetBrains/Exposed/pull/1771
+
+Breaking changes:
+* [Breaking changes](BREAKING_CHANGES.md)
```

Comment:
@bog-walk [This](https://github.com/JetBrains/Exposed/pull/1786) should be merged before.